ansible_runner/display_callback/callback/awx_display.py

```diff
@@ -348,6 +348,12 @@ def __init__(self):
         self.play_uuids = set()
         self.duplicate_play_counts = collections.defaultdict(lambda: 1)
 
+    def set_options(self, task_keys=None, var_options=None, direct=None):
```

Comment:
maybe it would be more robust to use `*args, **kwargs` to work with multiple versions. I see certain checks failed, but results expired.